dashboard/prisma/schema.prisma

```diff
@@ -13,17 +13,21 @@ datasource db {
 model Account {
   id                 String    @id @default(cuid())
   userId             String
-  providerType       String
-  providerId         String
+  type               String
+  provider           String
   providerAccountId  String
-  refreshToken       String?
-  accessToken        String?
-  accessTokenExpires DateTime?
+  refresh_token      String?
+  access_token       String?
+  expires_at         Int?
+  token_type         String?
+  scope              String?
+  id_token           String?
+  session_state      String?
```

Comment:
Should we be mixing letter casing? `userId` vs. `user_id`

Reply:
Yes, this model is used by the Prisma adapter. This model has changed from version 3 to version 4.
We had accidentally set it up originally using the version 3 model, which then failed when Oauth was setup.
I've taken this from https://authjs.dev/getting-started/adapters/prisma#schema > PostgreSQL
So the model needs to look like this, for it to work! But nice catch nonetheless! 😄

Reply:
It can be done using a custom Prisma adapter that maps the fields Auth.js expects to their camelCase counterparts, but this can be done in a separate PR as an enhancement 👌🏻

For the time being, maybe [this](https://authjs.dev/getting-started/adapters/prisma#naming-conventions) can clean things up a bit?

Reply:
Update - this sadly won't work with `@map` 😭 
Map only maps Prisma schema column definition to database columns name.
However, the Auth Prisma Adapter uses the Prisma definition...
Just looked at their example again. They don't actually change the casing of the Prisma model, just the casing for the actual DB schema, if that makes sense :/

<img width="624" height="426" alt="image" src="https://github.com/user-attachments/assets/bdc33747-086b-4b1a-b8fc-4a56b22fe1a9" />

So this seems to simply be used to ensure a consistent DB model schema.